From fb19a9a52f2126563b4e1fdcdeefca28d10b2444 Mon Sep 17 00:00:00 2001
From: Illia Vysochyn <bitfun6@gmail.com>
Date: Sun, 16 Oct 2022 21:02:27 +0200
Subject: [PATCH] Remove python bindings building

---
 setup.py | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/setup.py b/setup.py
index 04cf8925..091d0b9d 100644
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@ To package the wheel (after pip installing twine and wheel):
 To upload the binary wheel to PyPi
     twine upload dist/*.whl
 To upload the source distribution to PyPi
-    python setup.py sdist 
+    python setup.py sdist
     twine upload dist/dlib-*.tar.gz
 To exclude/include certain options in the cmake config use --yes and --no:
     for example:
@@ -119,7 +119,7 @@ class CMakeBuild(build_ext):
         except OSError:
             raise RuntimeError("\n*******************************************************************\n" +
                                   " CMake must be installed to build the following extensions: " +
-                               ", ".join(e.name for e in self.extensions) + 
+                               ", ".join(e.name for e in self.extensions) +
                                "\n*******************************************************************\n")
         return re.search(r'version\s*([\d.]+)', out.decode()).group(1)
 
@@ -138,7 +138,7 @@ class CMakeBuild(build_ext):
         cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
                       '-DPYTHON_EXECUTABLE=' + sys.executable]
 
-        cmake_args += cmake_extra_options 
+        cmake_args += cmake_extra_options
 
         cfg = 'Debug' if self.debug else 'Release'
         build_args = ['--config', cfg]
@@ -148,7 +148,7 @@ class CMakeBuild(build_ext):
             if sys.maxsize > 2**32:
                 cmake_args += ['-A', 'x64']
             # Do a parallel build
-            build_args += ['--', '/m'] 
+            build_args += ['--', '/m']
         else:
             cmake_args += ['-DCMAKE_BUILD_TYPE=' + cfg]
             # Do a parallel build
@@ -176,11 +176,11 @@ def num_available_cpu_cores(ram_per_build_process_in_gb):
     if 'TRAVIS' in os.environ and os.environ['TRAVIS']=='true':
         # When building on travis-ci, just use 2 cores since travis-ci limits
         # you to that regardless of what the hardware might suggest.
-        return 2 
+        return 2
     try:
-        mem_bytes = os.sysconf('SC_PAGE_SIZE') * os.sysconf('SC_PHYS_PAGES')  
+        mem_bytes = os.sysconf('SC_PAGE_SIZE') * os.sysconf('SC_PHYS_PAGES')
         mem_gib = mem_bytes/(1024.**3)
-        num_cores = multiprocessing.cpu_count() 
+        num_cores = multiprocessing.cpu_count()
         # make sure we have enough ram for each build process.
         mem_cores = int(floor(mem_gib/float(ram_per_build_process_in_gb)+0.5));
         # We are limited either by RAM or CPU cores.  So pick the limiting amount
@@ -228,9 +228,7 @@ setup(
     url='https://github.com/davisking/dlib',
     license='Boost Software License',
     ext_modules=[CMakeExtension('dlib','tools/python')],
-    cmdclass=dict(build_ext=CMakeBuild, test=PyTest),
     zip_safe=False,
-    tests_require=['pytest'],
     packages=['dlib'],
     keywords=['dlib', 'Computer Vision', 'Machine Learning'],
     classifiers=[
-- 
2.38.0

